hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/NameNodeExecutor.java

```diff
@@ -66,12 +72,20 @@ public void launchTask(final ExecutorDriver driver, final TaskInfo taskInfo) {
           .setTaskId(nameNodeTask.getTaskInfo().getTaskId())
           .setState(TaskState.TASK_RUNNING)
           .build());
+      healthCheckNN = new TimedHealthCheck(driver, nameNodeTask);
+      timer.scheduleAtFixedRate(healthCheckNN,
+        hdfsFrameworkConfig.getHealthCheckWaitingPeriod(),
```

Comment:
method chaining is a java style that we try to avoid as consistently as possible in this project. This is because it becomes harder to identify errors. It is a pretty common java style guideline.